api/src/main/java/jakarta/data/page/CursoredPage.java

```diff
@@ -51,62 +58,63 @@
+ * to this method must match the list of sorting criteria specified by
+ * {@link OrderBy} annotations of the repository method, {@link Sort}
+ * parameters of the page request, or <code>OrderBy</code> name pattern of
+ * the repository method. For example:</p>
  *
  * <pre>
  * Employee emp = ...
  * {@code PageRequest<Employee>} pageRequest = PageRequest.of(Employee.class)
  *                                                .size(50)
- *                                                .afterKeyset(emp.lastName, emp.firstName, emp.id);
+ *                                                .afterKey(emp.lastName, emp.firstName, emp.id);
  * page = employees.findByHoursWorkedGreaterThan(1500, pageRequest);
  * </pre>
  *
  * <p>By making the query for the next page relative to observed values,
- * not a numerical position, keyset pagination is less vulnerable to changes
+ * not a numerical position, key-based pagination is less vulnerable to changes
  * that are made to data in between page requests. Adding or removing entities
  * is possible without causing unexpected missed or duplicate results.
- * Keyset pagination does not prevent misses and duplicates if the entity
+ * Key-based pagination does not prevent misses and duplicates if the entity
  * properties which are the sort criteria for existing entities are modified
  * or if an entity is re-added with different sort criteria after having
  * previously been removed.</p>
  *
- * <h2>Keyset Pagination with &#64;Query</h2>
- *
- * <p>Keyset pagination involves generating and appending to the query
- * additional query conditions for the keyset properties. In order for that to
- * be possible, a user-provided
- * <a href="https://eclipse-ee4j.github.io/jakartaee-tutorial/#full-query-language-syntax">JPQL</a>
- * query must end with a
- * <code>WHERE</code> clause to which additional conditions can be appended.
- * Enclose the entire conditional expression of the <code>WHERE</code> clause
- * in parenthesis.
- * Sort criteria must be specified independently from the user-provided query,
- * either with the {@link OrderBy} annotation or
- * {@link Sort} parameters to {@link PageRequest}.
- * For example,</p>
+ * <h2>Key-based Pagination with {@code @Query}</h2>
+ *
+ * <p>Key-based pagination involves generating and appending additional
+ * restrictions involving the key fields to the <code>WHERE</code> clause of
+ * the query. For this to be possible, a user-provided JDQL or JPQL query must
+ * end with a <code>WHERE</code> clause to which additional conditions may be
+ * appended without otherwise changing the semantics of the query:</p>
+ * <ul>
+ * <li>The entire conditional expression of the <code>WHERE</code> clause should
+ * be enclosed in parentheses.
+ * <li>Sorting criteria must be specified independently of the user-provided
+ * query, either via the {@link OrderBy} annotation or, or by passing {@link Sort}
+ * criteria along with the {@linkplain PageRequest#sorts() page request}.
```

Comment:
Minor change in wording to avoid misinterpretation that you can supply `Sort` directly to your repository method for this pattern.
```suggestion
 * criteria within the {@linkplain PageRequest#sorts() page request}.
```